src/commands.def

```diff
@@ -1334,7 +1334,7 @@ commandHistory CLIENT_KILL_History[] = {
 {"8.0.0","`MAXAGE` option."},
 {"8.0.0","Replaced `master` `TYPE` with `primary`. `master` still supported for backward compatibility."},
 {"8.1.0","`ID` option accepts multiple IDs."},
-{"9.0.0","Added filters NAME, IDLE, FLAGS, LIB-NAME, LIB-VER, DB, CAPA, and IP."},
+{"9.0.0","Added filters NAME, IDLE, FLAGS, LIB-NAME, LIB-VER, DB, CAPA, and IP. And negative filters NOT-ID, NOT-MAXAGE, NOT-TYPE, NOT-ADDR, NOT-LADDR, NOT-USER, NOT-IDLE, NOT-FLAGS, NOT-NAME, NOT-LIB-NAME, NOT-LIB-VER, NOT-DB, NOT-CAPA, NOT-IP."},
```

Comment:
```suggestion
{"9.0.0","Added filters NAME, IDLE, FLAGS, LIB-NAME, LIB-VER, DB, CAPA, and IP. And negative filters NOT-ID, NOT-MAXAGE, NOT-TYPE, NOT-ADDR, NOT-LADDR, NOT-USER, NOT-IDLE, NOT-FLAGS, NOT-NAME, NOT-LIB-NAME, NOT-LIB-VER, NOT-DB, NOT-CAPA, NOT-IP."},
```
NOT-MAXAGE and NOT-IDLE are confusing with the NOT prefix, and I also don't see the use case. Why would you only want to kill clients that are active and/or only kill clients that are the youngest. We made some bad name choices in the past, so unless there is a strong reason to include them I would prefer to skip them.

Reply:
These two are only for symmetry and completeness of the filters. NOT-MAXAGE and NOT-IDLE are not ideal names, maybe MINAGE and ACTIVE would be better. We can revisit this discussion when there is actual user request in the future.